File: NotificationSystem/NotificationSystemWeb/Controls_Search/ctrSearch_Admin_Find.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_Admin_Find : System.Web.UI.UserControl
	{
        private string m_UserName = string.Empty;

        public string UserName
        {
            get
            {
                return m_UserName;
            }
            set
            {
                m_UserName = value;
            }
        }
        protected void Page_Load(object sender, EventArgs e)
		{
            m_UserName = txtUserName.Text;
        }
	}
}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Admin_Find.ascx.vb
Imports System
Imports System.Web.UI

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Admin_Find
        Inherits UserControl
        Private m_UserName As String = String.Empty

        Public Property UserName As String
            Get
                Return m_UserName
            End Get
            Set(value As String)
                m_UserName = value
            End Set
        End Property
        Protected Sub Page_Load(sender As Object, e As EventArgs)
            m_UserName = txtUserName.Text
        End Sub
    End Class
End Namespace
